scripts/setup-2.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+
```

Comment:
Can you add some more documentation to the top of this script regarding what it is for, what it does, etc?